ci/jjb/jobs/docs.yaml

```diff
@@ -126,7 +126,10 @@
             # create a virtualenv in which to install packages needed to build docs
             python3 -m venv --system-site-packages ~/docs_ve
             source ~/docs_ve/bin/activate
-            pip3 install celery sphinxcontrib-swaggerdoc 'django<2' django-filter djangorestframework djangorestframework-jwt drf-nested-routers psycopg2 sphinx git+https://github.com/snide/sphinx_rtd_theme.git@abfa98539a2bfc44198a9ca8c2f16efe84cc4d26 pyyaml virtualenv
+            pip3 install celery
```

Comment:
Celery is listed in docs/requirements.txt, it doesn't need to be listed separately here I don't think

Reply:
Oh cool, I missed it when I was looking at the list